spinalcordtoolbox/scripts/sct_qc.py

```diff
@@ -55,6 +56,11 @@ def get_parser():
                         help='If provided, this string will be mentioned in the QC report as the subject the process '
                              'was run on',
                         required=False)
+    parser.add_argument('-fps',
+                        metavar='float',
+                        type=float,
+                        help='The number of frames per second for output gif images',
+                        required=False)
```

Comment:
Ditto again for making it clear that `-fps` is only valid for the moco functions. 

(So, maybe add a note about this in the help descriptions, and throw an exception if combined with non-moco functions.)